src/Raven.Server/Documents/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -123,10 +124,37 @@ public DeleteExpiredDocumentsCommand(Dictionary<Slice, List<(Slice LowerId, Lazy
                 _database = database;
             }
 
-            public override int Execute(DocumentsOperationContext context)
+            protected override int ExecuteCmd(DocumentsOperationContext context)
             {
                 return DeletionCount = _database.DocumentsStorage.ExpirationStorage.DeleteExpiredDocuments(context, _expired);
             }
+
+            public override TransactionOperationsMerger.IReplayableCommandDto<TransactionOperationsMerger.MergedTransactionCommand> ToDto(JsonOperationContext context)
+            {
+                var keyValuePairs = new List<KeyValuePair<Slice, List<(Slice LowerId, LazyStringValue Id)>>>();
```

Comment:
We could use an Array here and call Array.Copy, rather than using list

Reply:
done